lib/cog/commands/alias/move.ex

```diff
@@ -1,4 +1,4 @@
-defmodule Cog.Commands.Alias.Mv do
+defmodule Cog.Commands.Alias.Move do
```

Comment:
This feels a little bit more like a rename, but I don't really feel that strongly about it.